lib/tree.R

```diff
@@ -65,9 +65,19 @@ read_in_metadata <- function(input, subject_identifier, label, feature_type, ran
   # feature of interest. Drop NA samples.
   metadata <- suppressMessages(readr::read_delim(file = input, delim = delim)) %>%
     dplyr::rename(., "subject_id" = subject_identifier) %>%
-    rename(., "feature_of_interest" = label) %>%
-    dplyr::filter(., !is.na(subject_id)) %>%
-    janitor::clean_names()
+    rename(., "feature_of_interest" = label)
+  
+  ## make check for NAs and warn user how many rows were dropped
+  original_row_count <- nrow(metadata)
+  metadata <- metadata %>% tidyr::drop_na() %>% janitor::clean_names()
```

Comment:
nit: might be worth doing the clean_names as part of the above block so this part is just about NAs, not sure if that's an ordered step though

Reply:
Nice, just made that change, good idea.